### DISCUSSION TITLE
search-for-good-ui

### DISCUSSION THREAD

**Discussion by Dharmik48** [TDEC-surat/TDEC-Register#23] [Feature Discussions]
Go to any preferrable site like [dribbble.com](https://dribbble.com), and look for some good  **minimal ui**

**Comment by Dharmik48** [comment]
I found [this one](https://dribbble.com/shots/16718697-Treecard), check out!

**Comment by meet41** [comment]
[Link 1](https://dribbble.com/shots/17473449-Analytics-Dashboard-UI-Concept?utm_source=Clipboard_Shot&utm_campaign=arshakir&utm_content=Analytics%20Dashboard%20UI%20Concept&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=arshakir&utm_content=Analytics%20Dashboard%20UI%20Concept&utm_medium=Social_Share)

[Link 2](https://dribbble.com/shots/15921854-School-Admin-Dashboard?utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share)

[Link 3](https://dribbble.com/shots/16000338-School-Admin-Dashboard-Attendance?utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard%20-%20Attendance&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard%20-%20Attendance&utm_medium=Social_Share)

[Link 4](https://dribbble.com/shots/6756879-Attendance-Dashboard?utm_source=Clipboard_Shot&utm_campaign=Riju&utm_content=Attendance%20Dashboard&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=Riju&utm_content=Attendance%20Dashboard&utm_medium=Social_Share)

**Reply by Dharmik48** [reply]
React with,
👍 to vote 1st design
👎 to vote 2nd design
😄 to vote 3rd design
🎉 to vote 4th design

**Reply by AkshDesai04** [reply]
Try `npm start` in terminal after cd'ing into the directory.

**Reply by Dharmik48** [reply]
So I guess [this design](https://dribbble.com/shots/15921854-School-Admin-Dashboard?utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share&utm_source=Clipboard_Shot&utm_campaign=azeroky&utm_content=School%20Admin%20-%20Dashboard&utm_medium=Social_Share) is the winner.

Look Nice

Only if we can pull it off in CSS😅

**Reply by AkshDesai04** [reply]
Well css is your territory m8😂

I'll try to contribute all I can but I can better contribute by overlooking things like the server, DB and some JS/HTML maybe

**Comment by meet41** [comment]
How to run site already installed node.js but then what

**Reply by Dharmik48** [reply]
I whatsapped you check out

**Comment by meet41** [comment]
Thanks, I'll check it out.
Yeah awesome 🤩 that I have already send in the link
We can also make it by our own if u want . 😅
